xonsh/tools.py

```diff
@@ -1560,8 +1560,18 @@ def __init__(self):
         self._alias_checksum = None
         self._path_mtime = -1
 
+    def __get_possible_names(self, key):
```

Comment:
I know, this was my intention but then I decided I don't really need a private method :)